apps/web/components/pages/kanban/menu-kanban-card.tsx

```diff
@@ -1,128 +1,271 @@
-import { useTeamMemberCard } from '@app/hooks';
+import { useOrganizationTeams, useTaskStatus, useTeamMemberCard, useTeamTasks } from '@app/hooks';
 import { activeTeamTaskId } from '@app/stores';
-import {
-  Popover,
-  PopoverContent,
-  PopoverTrigger
-} from '@components/ui/popover';
+import { Popover, PopoverContent, PopoverTrigger } from '@components/ui/popover';
 import { ThreeCircleOutlineVerticalIcon } from 'assets/svg';
 import { HorizontalSeparator, SpinnerLoader } from 'lib/components';
 import { PlanTask } from 'lib/features/task/task-card';
 import { useTranslations } from 'next-intl';
-import { useState } from 'react';
 import { useSetAtom } from 'jotai';
+import { ITeamTask, OT_Member } from '@app/interfaces';
+import { Combobox, Transition } from '@headlessui/react';
+import { Fragment, useCallback } from 'react';
+import { CheckIcon, ChevronUpDownIcon } from '@heroicons/react/20/solid';
 
-export default function MenuKanbanCard({
-  member,
-  item
-}: {
-  item: any;
-  member: any;
-}) {
-  const t = useTranslations();
-  const {
-    assignTask,
-    unassignTask,
-    assignTaskLoading,
-    unAssignTaskLoading
-  } = useTeamMemberCard(member);
-  const setActiveTask = useSetAtom(activeTeamTaskId);
-  const [load, setLoad] = useState<'' | 'assign' | 'unassign'>('');
-  const menu = [
-    {
-      name: t('common.EDIT_TASK'),
-      closable: true,
-      action: 'edit',
-      active: true,
-      onClick: () => {
-        setActiveTask({
-          id: item.id
-        });
-      }
-    },
-    {
-      name: t('common.ESTIMATE'),
```

Comment:
@CREDO23 why you removed this Estimate action?

Reply:
According to the issue description, we should not have it there, (#2413)

Reply:
@CREDO23 please add back. In the future, when you see that something is missing in the task description and that you want to REMOVE that functionality, always verify explicitly what to do with person who created the task (in Slack or in Task as comment etc) BEFORE you remove anything in the code! People who create issues can make mistakes too and we don't want some features to be removed by those mistakes.

Reply:
done @evereq